sdk/inc/azure/core/az_result.h

```diff
@@ -100,8 +100,6 @@ typedef enum
   AZ_ERROR_NOT_SUPPORTED = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 7),
 
   // Platform
-  AZ_ERROR_MUTEX = _az_RESULT_MAKE_ERROR(_az_FACILITY_PLATFORM, 1), ///< Mutex operation error.
```

Comment:
This was already removed in:
https://github.com/Azure/azure-sdk-for-c/pull/895

It shouldn't be showing up in the diff.